packages/dhis2/src/Adaptor.js

```diff
@@ -253,7 +251,7 @@ export function create(resourceType, data, options = {}, callback = s => s) {
     const { location } = response.headers;
     if (location) console.log(`Record available @ ${location}`);
 
-    return handleResponse(response, state, callback);
+    return handleResponse(response, state);
```

Comment:
I _think_ - and I'll come back to you on this - that in the main namespace here, we don't include `response` in the result. We just return to `state.data`.

The http methods should surface the response, but the resource helpers should not

Reply:
@josephjclark  What should we do here? I have fixed the other comments